HW13/requirements.txt

```diff
@@ -0,0 +1 @@
+webdriver-manager
```

Comment:
The main idea is to simplify management of binary drivers for different browsers, but here we are using Chrome only, it's not needed for now, but it can be useful in the future.